README.md

```diff
@@ -44,6 +44,25 @@ npm install <new-package>
 
 A SQL Server database server has been set up to run when the appeals service api is running.
 
+First, make sure you have a `.env.development` file in `./packages/appeals-service-api` and it has a `DATABASE_URL` environment variable defined with details pointing to your local database server (`mssql` Docker container).
```

Comment:
Do we need `.env.development` or just the `.env` one? If that is just an example, let's rename it `.env.example` to make that clear, and we could edit this paragraph?

> First, make sure you have a `.env` file in... (you can copy the `.env.example`)...